app/assets/stylesheets/cart.css.scss

```diff
@@ -1,30 +1,78 @@
 .cart {
-    background: white;
+  background:white;
 }
 
-.cart-picture {
-    background: transparent image-url("checkout.jpg") center;
+.cart h3 {
+  padding-top:10px;
```

Comment:
Properties should be ordered font-size, font-weight, padding-top, text-align<br>Colon after property should be followed by one space